src/rest/CommunityEndpoints.js

```diff
@@ -0,0 +1,61 @@
+import fetch from 'node-fetch'
+import { Contract } from 'ethers'
+
+import authFetch from './authFetch'
+
+const CommunityProduct = require('../../CommunityProduct.json')
+
+export async function joinCommunity(communityAddress, memberAddress, secret = undefined) {
+    const json = authFetch(
+        `${this.options.restUrl}/communities/${communityAddress}/joinRequests`,
+        this.session,
+        {
+            method: 'POST',
+            body: JSON.stringify({
+                memberAddress,
+                secret,
+            }),
+            headers: {
+                'Content-Type': 'application/json',
+            },
+        },
+    )
+    return json
+}
+
+export async function withdraw(communityAddress, memberAddress, wallet) {
+    const json = await authFetch(
```

Comment:
This call doesn't need to be authenticated? Looks like we could reuse the `memberStats` function here, right?